Add Vision Transformer + ViTFeatureExtractor (#10513)
huggingface/transformers: docs/source/model_doc/vit.rst

.. 
    Copyright 2020 The HuggingFace Team. All rights reserved.

    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
    the License. You may obtain a copy of the License at

        http://www.apache.org/licenses/LICENSE-2.0

    Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
    an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
    specific language governing permissions and limitations under the License.

Vision Transformer (ViT)
-----------------------------------------------------------------------------------------------------------------------

.. note::

    This is a recently introduced model so the API hasn't been tested extensively. There may be some bugs or slight

>     breaking changes to fix it in the future. Note that the API of :obj:`ViTFeatureExtractor` will change in the

Reviewer: ```suggestion
    breaking changes to fix it in the future. Note that the API of :class:`~transformers.ViTFeatureExtractor` will change in the
```

> Vision Transformer (ViT)

Reviewer: Why did you remove the copyright? Please put i back.

Author: Oh sorry, I thought it was a mistake. I didn't see this in tapas.rst, so deleted it.

-----------------------------------------------------------------------------------------------------------------------

.. note::

    This is a recently introduced model so the API hasn't been tested extensively. There may be some bugs or slight

>     breaking changes to fix them in the future. Note that the API of :obj:`ViTFeatureExtractor` will change in the
>     future, when more frameworks than just PyTorch will be supported.

Reviewer: ```suggestion
    breaking changes to fix it in the future. Note that the API of :obj:`ViTFeatureExtractor` might change in the
    future, when more frameworks than just PyTorch are supported.
```
